modules/auxiliary/gather/coldfusion_pms_servlet_file_read.rb

```diff
@@ -0,0 +1,114 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'CVE-2024-20767 - Adobe Coldfusion Arbitrary File Read',
+        'Description' => %q{
+          This module exploits an Improper Access Vulnerability in Adobe Coldfusion versions prior to version
+          '2023 Update 6' and '2021 Update 12'. The vulnerability allows unauthenticated attackers to request authentication
+          token in the form of a UUID from the /CFIDE/adminapi/_servermanager/servermanager.cfc endpoint. Using that
+          UUID attackers can hit the /pms endpoint in order to exploit the Arbitrary File Read Vulnerability.
+        },
+        'Author' => [
+          'ma4ter',          # Analysis & Discovery
+          'yoryio',          # PoC
+          'Christiaan Beek', # Msf module
+          'jheysel-r7'       # Msf module assistance
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2024-20767'],
+          ['URL', 'https://helpx.adobe.com/security/products/coldfusion/apsb24-14.html'],
+          ['URL', 'https://jeva.cc/2973.html'],
+
+        ],
+        'DisclosureDate' => '2024-03-12',
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [],
+          'SideEffects' => [IOC_IN_LOGS]
+        }
+      )
+    )
+
+    register_options(
+      [
+        Opt::RPORT(80),
```

Comment:
Default ColdFusion port is 8500/TCP. Maybe this would be a good idea to use it as a default value?